app/src/main/res/values/strings.xml

```diff
@@ -29,44 +29,45 @@
          For example, in Russian, Android's plural quantity "one" actually refers to "any number ending on 1 but not ending in 11".
          So while in English the extra non-plural form seems unnecessary duplication, it is necessary to give translators enough flexibility.
          In Catima, we use the plain string when meaning exactly 1, and otherwise use the plural forms -->
-    <string name="deleteTitle">Delete card</string>
+    <string name="deleteTitle">Deletion</string>
     <plurals name="deleteCardsTitle">
-        <item quantity="one">Delete <xliff:g>%d</xliff:g> card</item>
-        <item quantity="other">Delete <xliff:g>%d</xliff:g> cards</item>
+        <item quantity="one">Delete <xliff:g>%d</xliff:g> element</item>
+        <item quantity="other">Delete <xliff:g>%d</xliff:g> elements</item>
     </plurals>
-    <string name="deleteConfirmation">Delete this card permanently?</string>
+    <string name="deleteConfirmation">Delete this permanently?</string>
     <plurals name="deleteCardsConfirmation">
-        <item quantity="one">Delete this <xliff:g>%d</xliff:g> card permanently?</item>
-        <item quantity="other">Delete these <xliff:g>%d</xliff:g> cards permanently?</item>
+        <item quantity="one">Delete this <xliff:g>%d</xliff:g> element permanently?</item>
+        <item quantity="other">Delete these <xliff:g>%d</xliff:g> elements permanently?</item>
     </plurals>
     <!-- END NOTE: i18n oddness -->
     <string name="ok">OK</string>
     <string name="copy_to_clipboard">Copy ID to clipboard</string>
     <string name="share">Share</string>
     <string name="sendLabel">Send…</string>
-    <string name="editCardTitle">Edit Card</string>
-    <string name="addCardTitle">Add Card</string>
-    <string name="scanCardBarcode">Scan Card Barcode</string>
-    <string name="cardShortcut">Card Shortcut</string>
-    <string name="noCardsMessage">Add a card first</string>
-    <string name="card_ids_copied">Copied Card ID(s)</string>
-    <string name="barcodeImageDescriptionWithType">Image of card barcode of type <xliff:g>%s</xliff:g></string>
+    <string name="editCardTitle">Edit</string>
+    <string name="addCardTitle">Add</string>
+    <string name="scanCardBarcode">Scan Barcode</string>
+    <string name="cardShortcut">Shortcut</string>
+    <string name="noCardsMessage">Add something first</string>
```

Comment:
This happens when you open the card shortcut creator. Changing it like this makes the instructions incorrect because you need to create a card, not a group or so.